core/src/main/java/hudson/FilePath.java

```diff
@@ -2639,7 +2639,7 @@ public String validateAntFileMask(final String fileMasks, final int bound) throw
      * Default bound for {@link #validateAntFileMask(String, int, boolean)}.
      * @since 1.592
      */
-    public static int VALIDATE_ANT_FILE_MASK_BOUND = Integer.getInteger(FilePath.class.getName() + ".VALIDATE_ANT_FILE_MASK_BOUND", 10000);
+    public static final int VALIDATE_ANT_FILE_MASK_BOUND = Integer.getInteger(FilePath.class.getName() + ".VALIDATE_ANT_FILE_MASK_BOUND", 10000);
```

Comment:
My plan was to go through and fix all of the ones that were warned about as `should be final` (for which I'm not changing to `final`) to add a comment to warn that they shouldn't be finalized. If this is one of those, I can drop it and then when I get back, I'll include it w/ that comment. I didn't want to do that in this series because it'd be way too jittery.